frontend/src/TooljetDatabase/Table/index.jsx

```diff
@@ -89,6 +89,80 @@ const Table = ({ collapseSidebar }) => {
   const [isCellUpdateInProgress, setIsCellUpdateInProgress] = useState(false);
   const [referencedColumnDetails, setReferencedColumnDetails] = useState([]);
 
+  const [cachedOptions, setCahedOptions] = useState([]);
+
+  const fetchFkDataForColumn = (currentColumn, currentIndex) => {
```

Comment:
can we rename **currentColumn** to **foreignKey** if it is meant to be a foreign key